src/controllers/menu.controller.ts

```diff
@@ -61,10 +94,14 @@ export const updateMenu = async (req: Request, res: Response) => {
     }
 }
 
+
```

Comment:
Another new extra blank line. Try to lint files before committing if you remember. Thank you 🙏🏻

Reply:
Checked